C/Calcium/build_tarballs.jl

```diff
@@ -56,23 +54,23 @@ make install LIBDIR=$(basename ${libdir})
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = supported_platforms(; experimental=build_for_julia16_or_newer)
+platforms = supported_platforms()
 
 # The products that we will ensure are always built
 products = [
     LibraryProduct("libcalcium", :libcalcium)
 ]
 
 # Dependencies that must be installed before this package can be built
-f(v::VersionNumber) = build_for_julia16_or_newer ?  VersionNumber(v.major, v.minor, v.patch + 1) : v
+
 dependencies = [
-    Dependency("FLINT_jll"; compat = "~$(f(v"200.800.100"))"),
-    Dependency("Arb_jll", compat = "~$(f(v"200.2000"))"),
-    Dependency("Antic_jll", compat = "~$(f(v"0.200.500"))"),
-    Dependency("GMP_jll", build_for_julia16_or_newer ? v"6.2.0" : v"6.1.2"),
-    Dependency("MPFR_jll", build_for_julia16_or_newer ? v"4.1.1" : v"4.0.2"),
+    Dependency("FLINT_jll"; compat = "~$(v"200.800.101")"),
```

Comment:
@fingolfin Do we actually still need the `compat` entries? I always forget what is actually going on here.

Reply:
Yes we want those to make sure that only matching versions are used together. After all, this JLL won't be compatible with an older or much newer FLINT_JLL (it hopefully will be compatible with FLINT 200.800.200 etc. but probably not with 200.900.x)

Reply:
Thanks for the explanation!